sklearn/model_selection/tests/test_search.py

```diff
@@ -641,6 +641,56 @@ def test_refit():
     clf.fit(X, y)
 
 
+def test_refit_callable():
+    """
+    Test refit=callable, which adds flexibility in identifying the
+    "best" estimator.
+    """
+    def refit_callable(cv_results):
+        """
+        A dummy function tests `refit=callable` interface.
+        Return the index of a model that has the least
+        `mean_test_score`.
+        """
+        assert_true('mean_test_score' in cv_results)
```

Comment:
With the adoption of pytest, we are phasing out use of test helpers `assert_equal`, `assert_true`, etc. Please use bare `assert` statements, e.g. `assert x == y`, `assert not x`, etc.

Reply:
And I would actually assert that every key matches those in `cv_results_`, by fitting clf with refit=True before this and getting the list of keys (and also confirming that the `best_index_ != 0`.